ruby/README.md

```diff
@@ -0,0 +1,34 @@
+## Synopsis
+
+Ruby code examples for connecting to 6fusion API and interact with the different components of your account
+
+## Code structure
+
+As part of this repo, we provide the following examples:
+ - Meter Health verification
+
+
+## Installation
+
+Each tool require a different installation process.
+#### Ruby Script
+##### Requirements:
+
+- Ruby version: 2.1
+
+##### Installation instructions:
+  1. Download the repo
+  2.  Rename the file config/credentials.json.example to config/credentials.json
+    * Modify the file to add your credentials, and set the name of the report
+  3. Install ruby and bundler:
+    * Install curl : $ sudo apt-get install curl
```

Comment:
This instruction is debian based specific. Will be useful to include instructions for other famous distros like those based on RedHat (like CentOS).